src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -0,0 +1,87 @@
+package net.sf.jabref.logic.exporter;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import net.sf.jabref.BibDatabaseContext;
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.importer.ImportFormatPreferences;
+import net.sf.jabref.logic.importer.fileformat.BibtexImporter;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class ModsExportFormatTestFiles {
+
+    public BibDatabaseContext databaseContext;
+    public Charset charset;
+    public File tempFile;
+    public ModsExportFormat modsExportFormat;
+    public BibtexImporter testImporter;
+
+    public static final String PATH_TO_FILE = "src/test/resources/net/sf/jabref/logic/exporter/";
+
+    @Parameter
+    public String filename;
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+
+    @Parameters(name = "{0}")
+    public static Collection<String> fileNames() throws IOException {
+        try (Stream<Path> stream = Files.list(Paths.get(PATH_TO_FILE))) {
```

Comment:
Can you use class.getResource to get the resource folder? This makes the test more robust to changes in the project structure.